### ISSUE TITLE
Yahoo YUI https discontinued, can't load yui javascript files

### ISSUE THREAD

**Issue by gwirth79** [salesagility/SuiteCRM#3822]
Numerous files in SuiteCRM make use of externally loaded files from Yahoo at https://yui.yahooapis.com/
The use of https was done to fix the issue of mixed active content in web pages, since by default web browsers would block active mixed content.

This was resolved by Bug 3189, Fix# 719 which made all calls to the Yahoo site https.

Yahoo's https site is broken and won't be fixed. It has been down for several weeks already. The Yahoo YUI interfaces have been abandoned since 2014. Reliance on the use of the Yahoo site breaks functionality in SuiteCRM, often in subtle and undefined ways.
Recommend pulling all necessary files into SuiteCRM so that there are no external dependencies. This would also allow operation of SuiteCRM in a completely isolated environment, which is essential for secure locations and for testing purposes.

**Comment by chris001** [comment]
@shogunpol @GeorgeWL @gunnicom @croessler @jobvector
Issue #741 is an issue again.

The commit from #741 by @mattlorimer loads yui javascripts from yahoo `https`, which @gwirth79 reports is now broken : https://github.com/salesagility/SuiteCRM/commit/6b8300989ddab3cd5e0900c63dc4273251f73cce

Global search and replace `https://yui-s.yahooapis.com/2.5.0/build/` to instead load the the local copies of YUI2 and YUI3 from the site relative url, they're installed locally  in:
* `/include/javascript/yui/build/` 
* `/include/javascript/yui3/build/` for the yui3 urls.

**Comment by gunnicom** [comment]
Yes i once stated that all libraries should be local, so you dont need internet connection for running SuiteCRM. Cannot find where i wrote this some time ago.

**Comment by gunnicom** [comment]
Ah found it:
https://github.com/salesagility/SuiteCRM/issues/719

**Comment by GeorgeWL** [comment]
why was I mentioned in this?

I'm not a developer for this, I've only ever made a minor change to fix a minor bug to do with the PDF Export.
